Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2020-08 -- 4.6.0
+- [added] Added a new notification listening token refresh from Messaging and update its cache. (#6286)
```

Comment:
`its` is ambiguous. Do you mean the IID cache?

Reply:
Done. I updated the changelog to clarify it.